cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -1634,45 +1651,35 @@ def _parse_command_results(
 
         return {"status": status, "response": response, "error": error}
 
-    def action_run_command(self):
+    def _check_zombie_commands(self):
         """
-        Returns wizard action to select command and run it
+        Check commands that are running longer than the timeout
+        and mark them as finished
         """
-        context = self.env.context.copy()
-        context.update(
-            {
-                "default_server_ids": self.ids,
-            }
+        timeout = int(
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("cetmix_tower_server.command_timeout", 0)
```

Comment:
this `command_timeout` used only for zombie commands? maybe for any command without timeout we can use this parameter ...

Reply:
Probably we can do this is the future because currently we don't have a direct method to control command timeout.